src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx

```diff
@@ -432,10 +428,10 @@ export const ReviewCollapsible = ({
 
   return (
     <Collapsible
-      idPrefix={baseReviewLevel.title}
+      idPrefix={baseReviewLevel.titleParts.join('/')}
```

Comment:
using the slash here as id will be ok?

Reply:
Slash here is fine because we don't do any splitting on idPrefix